fossunited/scheduled_tasks.py

```
@@ -27,3 +27,43 @@
+        jobs.status = "Expired"
+
+        # try to mail them to inform
+        to_email = jobs.mail
+        cc_email = ["foundation@fossunited.org", "developers@fossunited.org"]
+        frappe.sendmail(
+            recipients=[to_email],
+            cc=cc_email,
+            subject=f"Job Posting Auto-Expiry Notice: {jobs.name} | {jobs.job_title}",
+            message=f"""
+            <p>Dear {jobs.company_name},</p>
+            <p>Your job posting titled <strong>{jobs.job_title}</strong> has not been updated in over 90 days and will now be automatically marked as <strong>Expired</strong>.</p>
```

Ruff (0.12.2)

Line too long (179 > 99)

(E501)